src/utils/contents.js

```diff
@@ -13,6 +13,11 @@ export function decodeContenthash(encoded) {
       decoded = contentHash.decode(encoded)
       const codec = contentHash.getCodec(encoded)
       if (codec === 'ipfs-ns') {
+
+        // convert the ipfs from base58 to base32 (url host compatible)
+        // if needed the hash can now be resolved through a secured origin gateway (<hash>.gateway.com)
+        decoded = contentHash.helpers.cidV0ToV1Base32(decoded)
```

Comment:
Unsure about context, but if this is needed for subdomain gateways, it should use `cidForWeb` from https://github.com/pldespaigne/content-hash/pull/43 instead